apps/web/vibes/soul/types/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './cart'
+export * from './filter'
+export * from './image'
+export * from './product'
+export * from './breadcrumbs'
+export * from './pagination'
+
+export type Brand = 'electric' | 'luxury' | 'warm'
```

Comment:
Shouldn't these be computed from the brands set up by the VIBE?

Reply:
Suggesting that we update this in a separate pr